test/plugin/featureaction/featureactionmanager.test.js

```diff
@@ -0,0 +1,78 @@
+goog.require('os.im.action.ImportActionCallbackConfig');
+goog.require('plugin.im.action.feature.Manager');
+
+describe('plugin.im.action.feature.Manager', function() {
+  const FeatureActionManager = goog.module.get('plugin.im.action.feature.Manager');
```

Comment:
`FeatureActionManager` isn't a module yet, so this isn't strictly necessary. However, if it works, then it does future proof us once we convert it to a module.